provisioner/roles/manage_ec2_instances/templates/instructor_inventory/instructor_inventory_rhel.j2

```diff
@@ -1,6 +1,7 @@
 [all:vars]
 {% if ssh_port is defined %}
 ansible_port={{ ssh_port }}
+ansible_ssh_private_key_file={{ playbook_dir }}/{{ ec2_name_prefix }}/{{ ec2_name_prefix }}-private.pem
```

Comment:
I'm not sure this will work except for on the node doing the provisioning. @IPvSean where does the instructor inventory end up in the lab?

Reply:
Instructor inventory and private key both end up located in {{playbook_dir}}/{{ec2_name_prefix}} on the node doing the provisioning, with the students getting their specific inventory and that same private key on their specific control node.

Reply:
Had a follow-up discussion on this with @willtome . We came to agree on what the final instructor file should look like...will make the change to adjust.